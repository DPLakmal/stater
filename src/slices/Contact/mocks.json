[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Get in Touch",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Have questions or need more information? Reach out to us.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "form_success_message": {
        "__TYPE__": "FieldContent",
        "value": "die",
        "type": "Text"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Eiusmod aute",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "map_iframe_url": {
        "__TYPE__": "LinkContent",
        "value": {
          "__TYPE__": "ExternalLink",
          "url": "https://slicemachine.dev"
        }
      },
      "recipient_email_address": {
        "__TYPE__": "FieldContent",
        "value": "shot",
        "type": "Text"
      }
    },
    "items": []
  }
]
