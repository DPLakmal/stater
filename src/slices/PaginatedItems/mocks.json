[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "number_of_items_per_page": {
        "__TYPE__": "FieldContent",
        "value": "82",
        "type": "Number"
      },
      "type": {
        "__TYPE__": "FieldContent",
        "value": "Blog",
        "type": "Select"
      }
    },
    "items": []
  }
]
