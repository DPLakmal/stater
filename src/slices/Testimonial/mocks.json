[
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "default",
    "primary": {
      "testimonials": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "image",
                {
                  "origin": {
                    "id": "main",
                    "url": "https://images.unsplash.com/photo-1587840171670-8b850147754e",
                    "width": 4480,
                    "height": 4480
                  },
                  "url": "https://images.unsplash.com/photo-1587840171670-8b850147754e?rect=0%2C667%2C4480%2C3146&w=47&h=33",
                  "width": 47,
                  "height": 33,
                  "edit": {
                    "zoom": 1,
                    "crop": {
                      "x": 0,
                      "y": 667
                    },
                    "background": "transparent"
                  },
                  "credits": null,
                  "alt": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ],
              [
                "description",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet. Amet minim mollit non deserunt ullamco est sit aliqua",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ],
              [
                "name",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "richerd",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ],
              [
                "designation",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "ceo",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "image",
                {
                  "provider": "imgix",
                  "url": "https://images.unsplash.com/photo-1616963738682-9df1389e5932?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHw3fHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzAwODk5MDZ8MA&ixlib=rb-4.0.3&q=85&rect=89,0,3346,2349&w=47&h=33",
                  "alt": "close up photo of green fruits",
                  "origin": {
                    "id": "XBMe5NM4ZHs",
                    "url": "https://images.unsplash.com/photo-1616963738682-9df1389e5932?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHw3fHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzAwODk5MDZ8MA&ixlib=rb-4.0.3&q=85",
                    "width": 3523,
                    "height": 2349
                  },
                  "width": 47,
                  "height": 33,
                  "edit": {
                    "background": "transparent",
                    "zoom": 1,
                    "crop": {
                      "x": 89,
                      "y": 0
                    }
                  },
                  "credits": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ],
              [
                "description",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet. Amet minim mollit non deserunt ullamco est sit aliqua",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ],
              [
                "name",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "sfda",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ],
              [
                "designation",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "asfafa",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "description",
                {
                  "__TYPE__": "StructuredTextContent",
                  "value": [
                    {
                      "type": "paragraph",
                      "content": {
                        "text": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet. Amet minim mollit non deserunt ullamco est sit aliqua",
                        "spans": []
                      },
                      "direction": "ltr"
                    }
                  ]
                }
              ],
              [
                "image",
                {
                  "provider": "imgix",
                  "url": "https://images.unsplash.com/photo-1603824231744-517ef271df7e?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwzfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzAwODk5MDZ8MA&ixlib=rb-4.0.3&q=85&rect=86,0,3518,2470&w=47&h=33",
                  "alt": "sliced lemon fruit on blue surface",
                  "origin": {
                    "id": "SDX4_TQcxGU",
                    "url": "https://images.unsplash.com/photo-1603824231744-517ef271df7e?crop=entropy&cs=srgb&fm=jpg&ixid=M3wzMzc0NjN8MHwxfHNlYXJjaHwzfHxjaXRydXMlMjBmcnVpdHxlbnwwfHx8fDE3MzAwODk5MDZ8MA&ixlib=rb-4.0.3&q=85",
                    "width": 3690,
                    "height": 2470
                  },
                  "width": 47,
                  "height": 33,
                  "edit": {
                    "background": "transparent",
                    "zoom": 1,
                    "crop": {
                      "x": 86,
                      "y": 0
                    }
                  },
                  "credits": null,
                  "__TYPE__": "ImageContent",
                  "thumbnails": {}
                }
              ]
            ]
          }
        ]
      },
      "top_padding": {
        "__TYPE__": "FieldContent",
        "type": "Select",
        "value": "Full"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "type": "Select",
        "value": "Full"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "dsfsdfsd",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "fsdfsdf",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "sdfsdfsdf"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Primary"
                }
              ]
            ]
          },
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Secondary"
                }
              ],
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "sdfsdfsdf"
                }
              ]
            ]
          }
        ]
      }
    },
    "items": []
  },
  {
    "__TYPE__": "SharedSliceContent",
    "variation": "variation2",
    "primary": {
      "top_padding": {
        "__TYPE__": "FieldContent",
        "value": "Half",
        "type": "Select"
      },
      "bottom_padding": {
        "__TYPE__": "FieldContent",
        "type": "Select",
        "value": "Full"
      },
      "subtitle": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Dorem ipsum",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "title": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Our Customer Feedback",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "description": {
        "__TYPE__": "StructuredTextContent",
        "value": [
          {
            "type": "paragraph",
            "content": {
              "text": "Corem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit interdum, ac aliquet odio mattis.",
              "spans": []
            },
            "direction": "ltr"
          }
        ]
      },
      "buttons": {
        "__TYPE__": "GroupContentType",
        "value": [
          {
            "__TYPE__": "GroupItemContent",
            "value": [
              [
                "label",
                {
                  "__TYPE__": "FieldContent",
                  "value": "camp",
                  "type": "Text"
                }
              ],
              [
                "icon_alignment",
                {
                  "__TYPE__": "FieldContent",
                  "value": "Right",
                  "type": "Select"
                }
              ],
              [
                "variant",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Select",
                  "value": "Primary"
                }
              ],
              [
                "icon_name",
                {
                  "__TYPE__": "FieldContent",
                  "type": "Text",
                  "value": "Person"
                }
              ]
            ]
          }
        ]
      },
      "place_id": {
        "__TYPE__": "FieldContent",
        "type": "Text",
        "value": "ChIJ5X3G32tQ4joRKYR_AnHdGNU"
      }
    },
    "items": [
      {
        "__TYPE__": "GroupItemContent",
        "value": []
      }
    ]
  }
]
