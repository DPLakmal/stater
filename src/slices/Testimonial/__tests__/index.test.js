import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import mocks from "../mocks.json";
import Component from "../index";
import simplifyMockData from "@/utils/prismic/simplifyMockData";

// Mock Swiper and its dependencies
jest.mock("swiper/react", () => ({
  Swiper: ({ children }) => <div>{children}</div>,
  SwiperSlide: ({ children }) => <div>{children}</div>,
}));

// Mock EmblaCarousel and its dependencies
jest.mock("embla-carousel-react", () => ({
  EmblaCarousel: ({ children }) => <div>{children}</div>,
}));

jest.mock("swiper/css", () => jest.fn());
jest.mock("swiper/css/navigation", () => jest.fn());
jest.mock("swiper/modules", () => jest.fn());

jest.mock("embla-carousel-react", () => ({
  __esModule: true,
  default: jest.fn(() => [jest.fn(), jest.fn()]),
}));

describe("Testimonial Component", () => {
  // top padding
  it("first variation with full top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "Full";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation with half top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "Half";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation without top padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.top_padding = "None";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  // bottom padding
  it("first variation with full bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "Full";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation with half bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "Half";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });
  it("first variation without bottom padding renders correctly", () => {
    let data = simplifyMockData(mocks[0]);
    data.primary.bottom_padding = "None";
    const { container } = render(<Component slice={data} />);
    expect(container).toMatchSnapshot();
  });

  // it("second variation renders correctly", () => {
  //   const data = simplifyMockData(mocks[1]);
  //   console.log(typeof data);
  //   expect(1 + 1).toBe(2);
  //   const { container } = render(<Component slice={{}} />);
  //   // expect(container).toMatchSnapshot();
  // });
});
