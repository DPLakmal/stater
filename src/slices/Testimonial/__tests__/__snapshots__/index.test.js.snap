// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial Component first variation with full bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pt-[40px] md:pt-[80px] xl:pt-[120px] pb-[40px] md:pb-[80px] xl:pb-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Testimonial Component first variation with full top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pt-[40px] md:pt-[80px] xl:pt-[120px] pb-[40px] md:pb-[80px] xl:pb-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Testimonial Component first variation with half bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pb-[20px] md:pb-[40px] xl:pb-[60px] pt-[40px] md:pt-[80px] xl:pt-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Testimonial Component first variation with half top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pt-[20px] md:pt-[40px] xl:pt-[60px] pb-[40px] md:pb-[80px] xl:pb-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Testimonial Component first variation without bottom padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pb-0 pt-[40px] md:pt-[80px] xl:pt-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`Testimonial Component first variation without top padding renders correctly 1`] = `
<div>
  <section
    data-slice-variation="default"
  >
    <div
      class="px-[16px] md:px-[40px] xl:px-[80px] pt-0 pb-[40px] md:pb-[80px] xl:pb-[120px] "
    >
      <div
        class="mx-auto w-full max-w-[1280px] "
      >
        <div
          class="flex flex-col  items-center xl:mx-[88px]"
        >
          <div
            class="text-center  "
          >
            <div
              class="mt-[10px] mb-[10px] md:mt-[20px]"
            />
            <div
              class="flex flex-row justify-center gap-x-[20px] mt-[20px] md:mt-[30px]"
            >
              <a
                class=" bg-black h-fit btn btn-primary"
                href=""
              >
                sdfsdfsdf
              </a>
              <a
                class=" bg-black h-fit btn bg-secondary"
                href=""
              >
                sdfsdfsdf
              </a>
            </div>
          </div>
          <div
            class="flex items-center justify-center w-full h-full mt-[30px] md:mt-[40px] xl:mt-[60px]"
          >
            <div
              class="flex items-center justify-center w-full h-full overflow-hidden"
            >
              <div
                class="flex embla__container "
              >
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                    <p
                      class=" mb-7 last:mb-0"
                    />
                  </div>
                </div>
                <div
                  class="embla__slide w-[100%] mr-[28px] lex flex-col items-start bg-[#F8F8F8] max-w-[354px] px-[26px] py-[20px] md:py-[30px]  min-w-[calc(50%-10px)] lg:min-w-[calc(33.33%-12px)]"
                >
                  <p
                    class=" mb-7 last:mb-0"
                  />
                  <div
                    class="flex flex-col"
                  >
                    <div
                      class="text-body-large mb-[10px]"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
